azure-pipelines.yml

```diff
@@ -89,22 +96,6 @@ jobs:
         MSBuildConfiguration: 'Release'
         vmImage: 'vs2017-win2016'
         UsesOpenCV: 'FALSE'
-      VS2017_x86-msvc_debug_msbuild:
```

Comment:
We should validate these changes with the packaging build too, We are getting ready to hand this over it would be nice to know this won't be a problem for them in several month when a new release gets planned

Reply:
All of these builds that we no longer run are not run on the packaging build either. The packaging build just runs:

32-bit and 64-bit builds using VS2017 and a Ninja generator